tests/utils/test_dates.py

```diff
@@ -94,3 +94,12 @@ def test_negative_num_given(self):
 
             for i in range(num):
                 self.assertTrue(timezone.is_localized(result[i]))
+
+    def test_delta_cron_presets(self):
+        num = 2
+        for delta in dates.cron_presets:
+            result = dates.date_range(datetime(2016, 1, 1), num=num, delta=delta)
+               self.assertEqual(len(result), num)
```

Comment:
Can you make it more descriptive? You don't have to test all cases. One is enough if it is tested thoroughly.
https://testing.googleblog.com/2019/12/testing-on-toilet-tests-too-dry-make.html